recipes-kernel/kernel-tests/kernel-test-fbomb-files/run-ptest

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./fbomb -d 24h
```

Comment:
Recommend that you add an `exec 2>&1` call here to keep your stderr and stdout lines synchronized. [See here](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/3560/Yocto-PTests?anchor=gotchya-%23-1%3A).